datalad/support/external_versions.py

```diff
@@ -252,6 +255,43 @@ def keys(self):
     def __contains__(self, item):
         return bool(self[item])
 
+    def add(self, name, func=None, interesting=True):
+        """Add a version checker
+
+        Could be used e.g. by extensions to add important to them custom checks.
```

Comment:
This took me a few attempts to parse, particularly the "to them" part.  Perhaps "This method allows third-party libraries to define additional checks"?